src/components/attendee/Banner/Banner.module.scss

```diff
@@ -0,0 +1,36 @@
+.banner {
+  position: fixed;
+  z-index: 90;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  opacity: 1;
+
+  transition: opacity 1.5s ease;
+  transition-delay: 1s;
+  height: 100vh;
+  width: 100vw;
+}
+
+.bannerContent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  color: white;
+
+  height: 100%;
+  width: 100%;
+}
+
+.actionButtons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.closeButton {
+  padding: 0 12px;
```

Comment:
It is easier to manage uniforming of spacing if these hard-coded values are replaced with functions like `space.empty()` and `space.gap()`

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2952/commits/31283fe4933ee54a28e2a97af203f53d7c7886bf